### PR TITLE
Move foreign key for inventory ids

### LINKED ISSUE #21: Items and products can hold only one inventory id each
You should be able to apply more than one inventory id to items and products each

### PR DESCRIPTION
**Pull request by clerie**
Fixes #21

### CODE REVIEW

**Comment by clerie** [comment]
I recomment waiting for #20 to get mergend before generating migrations.

**Comment by clerie** [comment]
Also we should discuss, how we want to handle the migrations. Django want us to define a default for the new required `ForeignKey`, which is not an optimal. I would prefer deleting all existing entries.

**Review by e1mo — APPROVED** [review]
Looks good to me

**Review by em0lar — APPROVED** [review]

**Review by e1mo — CHANGES_REQUESTED** [review]
Oh wait, I just saw that the migration from #20 was deleted and the blank options are not represented in the models.

See https://github.com/chaosinventory/chaosinventory/compare/clerie%2Finventory-ids...clerie%2Fmodel-blanks?diff=unified

Or was I just looking wrong?

**Comment by clerie** [comment]
> Oh wait, I just saw that the migration from #20 was deleted and the blank options are not represented in the models.

Ok, did a rebase and redone the migrations stuff. Hope its now ok.

**Comment by margau** [comment]
It seems that we have the lint failing. Interestingly, that happens in a line that was not touched?

File: src/chaosinventory/base/models.py [hunk 1]
```diff
@@ -92,6 +92,10 @@ class ProductData(CommonTypeData):
 
 
 class ProductInventoryId(CommonInventoryId):
+    product = models.ForeignKey(
+        'Product',
+        on_delete=models.CASCADE,
+    )
```
Comment (e1mo) [inline src/chaosinventory/base/models.py:98]:
```suggestion
    )
    
```

File: src/chaosinventory/base/models.py [hunk 2]
```diff
@@ -104,6 +108,10 @@ class ItemData(CommonTypeData):
 
 
 class ItemInventoryId(CommonInventoryId):
+    item = models.ForeignKey(
+        'Item',
+        on_delete=models.CASCADE,
+    )
```
Comment (e1mo) [inline src/chaosinventory/base/models.py:114]:
```suggestion
    )
    
```

**Review by e1mo — CHANGES_REQUESTED** [review]
Looks good so far, thanks! However, I would be happier if the CI would be happy (:

See here for the full CI log: <https://github.com/chaosinventory/chaosinventory/pull/24/checks?check_run_id=1998086234>

**Review by margau — APPROVED** [review]

**Review by e1mo — APPROVED** [review]

**Review by adnidor — APPROVED** [review]

**Pull request merged**